tools/musescore_crashreporter/mainwindow.cpp

```diff
@@ -0,0 +1,347 @@
+//=============================================================================
+//  MuseScore
+//  Music Composition & Notation
+//
+//  Copyright (C) 2017 Nikolaos Hatzopoulos (nickhatz@csu.fullerton.edu)
+//
+//  This program is free software; you can redistribute it and/or modify
+//  it under the terms of the GNU General Public License version 2.
+//
+//  This program is distributed in the hope that it will be useful,
+//  but WITHOUT ANY WARRANTY; without even the implied warranty of
+//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+//  GNU General Public License for more details.
+//
+//  You should have received a copy of the GNU General Public License
+//  along with this program; if not, write to the Free Software
+//  Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
+//=============================================================================
+
+#include "mainwindow.h"
+#include "ui_mainwindow.h"
+
+//-----------------------------------------------------------------------------
+// launcher(program) : executes a file with filepath: program
+// this function is only valid under windows Operating System
+// when the executable is executed the process is send to the background
+//-----------------------------------------------------------------------------
+
+bool launcher(wstring program)
```

Comment:
This one can probably be made more cross platform. See QProcess::startDetached http://doc.qt.io/qt-5/qprocess.html#startDetached